Emojigrams/Emojigrams/EmojigramGuesserBaseViewController.swift

```diff
@@ -70,5 +77,11 @@ class EmojigramGuesserBaseViewController: UIViewController {
         alertC.addAction(okAction)
         self.present(alertC, animated: true, completion: nil)
     }
+}
 
+extension EmojigramGuesserBaseViewController: UITextFieldDelegate {
```

Comment:
#pp should we make it an extension? or put it in the base class?

Reply:
I like using Extensions as another way to keep code organized, so you know that all your UITextFieldDelegate methods are in this one spot. This is especially useful if there's a lot of optional methods that can be implemented rather than just other sort of protocols that has no big methods to implement.